mlrun/run.py

```diff
@@ -987,6 +987,14 @@ def run_pipeline(
     arguments = arguments or {}
 
     if remote or url:
+        from .projects.pipelines import pipeline_context
```

Comment:
I think you should be setting
`pipeline_context.workflow.engine="kfp"` instead of `pipeline_context.workflow = True` because workflow is not meant to be a boolean and we should strive to preserve variables types